Assets/HoloToolkit/Input/Tests/Scripts/DisplaySpeechKeywords.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace HoloToolkit.Unity.InputModule.Tests
+{
+    public class DisplaySpeechKeywords : MonoBehaviour
+    {
+        public SpeechInputSource speechInputSource;
+        public Text textPanel;
```

Comment:
Nit: use Text Mesh instead of uGUI.  Although really it doesn't matter since it's a test scene.